ui/component/fileRenderDownload/view.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React from 'react';
```

Comment:
New view for downloadable content. Download only content does not go through `<FileRenderInitiator>` (previous ViewerInitiator)